sky/compositor/picture_layer.cc

```diff
@@ -3,11 +3,17 @@
 // found in the LICENSE file.
 
 #include "sky/compositor/picture_layer.h"
+
 #include "base/logging.h"
+#include "sky/compositor/checkerboard.h"
+#include "sky/compositor/raster_cache.h"
 
 namespace sky {
 namespace compositor {
 
+// TODO(abarth): Make this configurable by developers.
+const bool kCheckerboardRasterizedLayers = false;
```

Comment:
maybe rename this something with "debug" in its name to clarify its role as a debugging aid rather than e.g. the fallback behavior that browsers use when they can't paint a tile fast enough?